nss-ice-phantoms/WorkforceManagement
# Training Programs - Details, Edit

# Description
Completed Details and Edit for Training Programs.

## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout meg-training-details-edit`
1. Run BangazonWorkforce
1. Navigate to **Training Programs** in the browser
1. Verify that the **Details** page works, and shows _name, start date, end date, max number of attendees, and a list of attendees_. If you do not see any attendees, you will need to insert them into your database. Here's some testing data! (Note: You may need to change Ids).
   ```
   insert into TrainingProgram ([Name], StartDate, EndDate, MaxAttendees) 
   values ('Super Cool Training', '14 April 2019', '15 April 2019', 10)
   insert into TrainingProgram ([Name], StartDate, EndDate, MaxAttendees) 
   values ('Learn How To Do Things!', '14 May 2019', '15 Feb May', 10)
   insert into TrainingProgram ([Name], StartDate, EndDate, MaxAttendees) 
   values ('Intro to HTML', '14 June 2019', '15 Feb June', 10)
   insert into EmployeeTraining (EmployeeId, TrainingProgramId) values (1, 1)
   insert into EmployeeTraining (EmployeeId, TrainingProgramId) values (2, 2)
   insert into EmployeeTraining (EmployeeId, TrainingProgramId) values (3, 3)
   ```
1. Click "Edit" on the details page and verify that you can edit a program.


# Checklist:
